### PR TITLE
Examples: Clean up

### PR DESCRIPTION
**Pull request by WestLangley**
Follow-on to #21969, which removed the `premultipliedAlpha` test from `webgl_materials_physical_transmission.html`.

### CODE REVIEW

File: examples/webgl_materials_physical_transmission.html [hunk 1]
```diff
@@ -65,7 +65,6 @@
 				renderer.outputEncoding = THREE.sRGBEncoding;
 
 				scene = new THREE.Scene();
-				scene.background = hdrEquirect;
```
Comment (WestLangley) [inline examples/webgl_materials_physical_transmission.html:68]: Note: background was  not removed. There were two backgrounds specified.

**Comment by WestLangley** [comment]
I am not a fan of these examples that rotate the object unnecessarily. They serve only to obfuscate potential problems.

I'd recommend a GUI to toggle-off the animation -- resetting the rotation to zero.

**Comment by WestLangley** [comment]
Updated screen shot is not correct... Not sure what is going on...

webgl_materials_physical_transmission

**Comment by Mugen87** [comment]
> Updated screen shot is not correct... Not sure what is going on...

Have you tried `npm i` before making the screenshot?

**Comment by WestLangley** [comment]
@Mugen87 I did, and the generated screenshot is still not correct. Can you give it a try?

**Comment by mrdoob** [comment]
@WestLangley can you try running `npm i` inside the `tets/` folder?

**Comment by mrdoob** [comment]
Thanks!

**Comment by WestLangley** [comment]
>@WestLangley can you try running npm i inside the test/ folder?

Thanks! That fixed it.

**Pull request merged**